File: samples/snippets/visualbasic/VS_Snippets_Wpf/ListViewCustomView/visualbasic/myview.vb
Imports System
Imports System.Windows
Imports System.Windows.Controls
Imports System.Windows.Controls.Primitives
Imports System.Windows.Data
Imports System.Globalization
Imports System.ComponentModel
Imports System.Windows.Media
Imports System.Windows.Shapes
Imports System.Xml

Namespace SDKSample
	Public Class CoolView
		Inherits ViewBase
		Protected Overrides ReadOnly Property DefaultStyleKey() As Object
		  Get
			Return New ComponentResourceKey(Me.GetType(), "MyViewDSK")
		  End Get
		End Property

		Protected Overrides ReadOnly Property ItemContainerDefaultStyleKey() As Object
		  Get
			Return New ComponentResourceKey(Me.GetType(), "MyViewItemDSK")
		  End Get
		End Property
	End Class
End Namespace

File: samples/snippets/csharp/VS_Snippets_Wpf/ListViewCustomView/CSharp/MyView.cs
using System;
using System.ComponentModel;
using System.Globalization;
using System.Windows;
using System.Windows.Controls;
using System.Windows.Controls.Primitives;
using System.Windows.Data;
using System.Windows.Media;
using System.Windows.Shapes;
using System.Xml;

namespace SDKSample
{
    public class CoolView : ViewBase
    {
        protected override object DefaultStyleKey
        {
            get
            {
                return new ComponentResourceKey(this.GetType(), "MyViewDSK");
            }
        }

        protected override object ItemContainerDefaultStyleKey
        {
            get
            {
                return new ComponentResourceKey(this.GetType(), "MyViewItemDSK");
            }
        }
    }
}
